test/linalg/symmetric.jl

```diff
@@ -153,15 +154,24 @@ end
 
             @testset "isposdef[!]" begin
                 @test isposdef(Symmetric(asym))  == isposdef(asym)
+                @test isposdef(Symmetric(aposs)) == isposdef(aposs) == true
                 @test isposdef(Hermitian(aherm)) == isposdef(aherm)
                 @test isposdef(Hermitian(apos))  == isposdef(apos) == true
                 if eltya != Int #chol! won't work with Int
                     @test isposdef!(Symmetric(copy(asym)))  == isposdef(asym)
+                    @test isposdef!(Symmetric(copy(aposs))) == isposdef(aposs) == true
                     @test isposdef!(Hermitian(copy(aherm))) == isposdef(aherm)
                     @test isposdef!(Hermitian(copy(apos)))  == isposdef(apos) == true
                 end
             end
 
+            @testset "$f" for f in (det, logdet, logabsdet)
+                @test all(f(apos) .≈ f(Hermitian(apos, :U)))
+                @test all(f(apos) .≈ f(Hermitian(apos, :L)))
+                @test all(f(aposs) .≈ f(Symmetric(aposs, :U)))
```

Comment:
right but if things were to accidentally dispatch to cholesky this wouldn't identify it